generate-src-docs.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+rm -rf src-docs
+# There is a problem with lazydocs and using the flask Blueprint class which is causing an error,
+# skip that file
+PYTHON_FILES=$(find repo_policy_compliance -name "*.py" -not -path "**/blueprint.py")
+# The word splitting is ententional here
```

Comment:
/s/ententional/intentional

Reply:
that was ententional too 😉  should be fixed shortly